### DISCUSSION TITLE
Reporting status of failed actions after merge?

### DISCUSSION THREAD

**Discussion by michaelfig** [chdsbd/kodiak#787] [Q&A]
Hi, I have a self-hosted Kodiak install that's working well for me.

There's just one thing: I have several Github Actions that run on push to master/main, rather than as part of the PR.  I can't find out where the actions failure notifications are going.  It seems that they are targeted at my Kodiak Github App bot account, but I'd like to have them go to our team, since it means the main branch is not green (should not block merging of the PR, but should be noticed).

Do you have any advice on how I could configure Kodiak to announce these failures to our team?

**Comment by sbdchd** [comment]
Curious what you mean by, "they are targeted at my Kodiak Github App bot account"?

I think github supports sending email notifications for failing builds with github actions.
There are probably web hooks for actions statuses if you wanted to hook something up manually


Edit: Oh does the github actions being targeted at the Kodiak Github App bot account mean that the email notifications that would normally take place don't go through? A little hacky, but maybe some automated email forwards to a team email would work? Or swapping out the github bot account email with a team mailing list?

**Comment by chdsbd** [comment]
I think you could use the GitHub Slack app and GitHub scheduled reminders to receive notifications for failed builds
